R/object-import.R

```diff
@@ -34,3 +39,19 @@ format.rd_section_reexport <- function(x, ...) {
     "\n}}\n"
   )
 }
+
+find_topic_in_package_reexp <- function(pkg, topic) {
+  path <- tryCatch(
```

Comment:
It feels like this pattern is repeated in a couple of places, so might be worth extracting out.

Reply:
Yeah.